libraries/llvm/types.c

```diff
@@ -5,11 +5,19 @@
 typedef int64_t Int;
 typedef double Double;
 
+struct Header {
+  uint64_t rc;
+  void (*eraser)(void *);
+};
```

Comment:
Nice! Now does `buffer.c` also need this? (if yes, please don't fix this in this PR, since this would heavily conflict with #469 )

Reply:
Yes.